core/controller/src/main/scala/org/apache/openwhisk/core/controller/Packages.scala

```diff
@@ -299,9 +299,14 @@ trait WhiskPackagesApi extends WhiskCollectionAPI with ReferencedEntities {
       case b: Binding =>
         val docid = b.fullyQualifiedName.toDocId
         logging.debug(this, s"fetching package '$docid' for reference")
-        getEntity(WhiskPackage.get(entityStore, docid), Some {
-          mergePackageWithBinding(Some { wp }) _
-        })
+        if (docid == wp.docid) {
+          logging.error(this, "unexpected package binding refers to itself: $docid")
```

Comment:
this won't interpolate correctly - you're missing an s"..." in the log message. 
same below.

Reply:
pushed a commit to fix.